google-cloud-storage/src/test/java/com/google/cloud/storage/PackagePrivateMethodWorkarounds.java

```diff
@@ -59,11 +60,26 @@ public static Blob blobCopyWithStorage(Blob b, Storage s) {
     return new Blob(s, builder);
   }
 
-  public static Function<WriteChannel, Optional<StorageObject>> maybeGetStorageObjectFunction() {
+  public static Function<WriteChannel, Optional<BlobInfo>> maybeGetStorageObjectFunction() {
```

Comment:
nit: change the method name
```suggestion
  public static Function<WriteChannel, Optional<BlobInfo>> maybeGetBlobInfoFunction() {
```

Reply:
Good catch, fix pushed.